cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,85 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should handle pagination', () => {
+    cy.get('.-next').should('exist');
+
+    cy.get('.-previous').should('exist');
+  });
+
+  it('should handle rows count selection', () => {
+    cy.get('select[aria-label="rows per page"]')
+      .should('exist')
+      .select('5 rows');
+
+    cy.get('select[aria-label="rows per page"]')
+      .should('have.value', '5');
+
+    cy.get('select[aria-label="rows per page"]')
+      .should('exist')
+      .select('20 rows');
+
+    cy.get('select[aria-label="rows per page"]')
+      .should('have.value', '20');
+  });
+
+  it('should add a new worker', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type('John');
+    cy.get('#lastName').type('Doe');
+    cy.get('#userEmail').type('john.doe@mail.com');
+    cy.get('#age').type('30');
+    cy.get('#salary').type('50000');
+    cy.get('#department').type('Engineering');
+    cy.get('#submit').click();
+
+    cy.contains('John');
+  });
+
+  it('should delete a worker', () => {
+    cy.contains('Kierra').parent()
+      .find('[title="Delete"]').click();
+
+    cy.contains('Kierra').should('not.exist');
+  });
+
+  it('should delete all workers', () => {
+    cy.get('[title="Delete"]').eq(0)
+      .click();
+
+    cy.get('[title="Delete"]').eq(0)
+      .click();
+
+    cy.get('[title="Delete"]').eq(0)
+      .click();
+
+    cy.get('.rt-tbody').should('not.contain', 'Cierra');
```

Comment:
what if they will have different names? Find another way to check that they are deleted